daemon/cmd/datapath.go

```diff
@@ -352,10 +353,9 @@ func (d *Daemon) initMaps() error {
 		}
 	}
 
-	pm := probes.NewProbeManager()
-	supportedMapTypes := pm.GetMapTypes()
+	lruHashErr := features.HaveMapType(ebpf.LRUHash)
 	createSockRevNatMaps := option.Config.EnableSocketLB &&
-		option.Config.EnableHostServicesUDP && supportedMapTypes.HaveLruHashMapType
+		option.Config.EnableHostServicesUDP && lruHashErr == nil
```

Comment:
nit: Do we need extra var? maybe just fold it in: `&& features.HaveMapType(ebpf.LRUHash) == nil`?